secret-sharing/src/suites/p384.rs

```diff
@@ -7,10 +7,6 @@ use p384::{
 
 use super::{FieldDigest, GroupDigest};
 
-/// Domain separation tag for encoding to NIST P-384 prime field or curve
-/// using the SHA3-384 hash function.
-const NIST_P384_SHA3_384_ENC_DST: &[u8] = b"P384_XMD:SHA3-384_SSWU_RO_";
```

Comment:
Will not using standard DSTs make it harder for third-party implementations?

Reply:
I guess if DSTs don't use something like length-prefix encoding for the components (e.g. like TupleHash), then it doesn't make it any easier as soon as there are custom DSTs. So in this case it is fine and this can be ignored.

Reply:
By harder for third party implementations you mean that it would be great to have standard dst in the package so that third party apps can easily construct full dst?

IEFT draft _hashing to elliptic curves_ suggestions name `<app>-V<xx>-CS<yy>-<suiteID>-<enc>`  where `<xx>` and `<yy>` are two-digit numbers indicating the version and ciphersuite, respectively, and `<suiteID>` is the Suite ID of the encoding used in ciphersuite `<yy>`. I'm not sure what `<yy>` should be, but I guess in our case the dst should be `oasis-core/keymanager/churp: V00-P384_XMD:SHA3-384_SSWU_RO_-<runtime_id>-<churp_id>-shareholder` or `CHURP-V00-P384_XMD:SHA3-384_SSWU_RO_-<runtime_id>-<churp_id>-shareholder`. However, if we want to be consistent with our signature ctx, the tag should be `oasis-core/keymanager/churp: sharehoder for runtime <id> for churp <id>`. Not sure what is better.

Reply:
> length-prefix encoding for the components

Tested locally, and I get the same result. I also checked the code, and length prefix or suffix (not sure what) is added only to the full dst (concatenated components).